packages/typespec-python/scripts/eng/pyrightconfig.json

```diff
@@ -2,5 +2,6 @@
     "reportUnnecessaryCast": "warning",
     "reportTypeCommentUsage": true,
     "reportMissingImports": false,
+    "reportAttributeAccessIssue": false,
```

Comment:
After https://github.com/microsoft/typespec/issues/7132 fixed, we shall remove this line.